main.go

```diff
@@ -173,6 +186,9 @@ func run(ctx context.Context, ms *xmain.State) (err error) {
 
 	var pw png.Playwright
 	if filepath.Ext(outputPath) == ".png" {
+		if *darkThemeFlag != math.MaxInt64 {
+			return xmain.UsageErrorf("--dark_theme cannot be used while exporting to another format other than .svg")
```

Comment:
this should just be a warning. allow compilation and just warn that dark theme is ignored

Reply:
Oh, true, but how can I print a warning? Something like `ms.Log.Warn.Printf` ?

Reply:
yeup https://github.com/terrastruct/d2/blob/ea5db1d424e0d1c635ff1a799d4c8172f86660f1/watch.go#L382